examples/fungible-token/src/lib.rs

```diff
@@ -111,8 +111,9 @@ impl FungibleToken {
     /// Initializes the contract with the given total supply owned by the given `owner_id`.
     #[init]
     pub fn new(owner_id: AccountId, total_supply: U128) -> Self {
```

Comment:
Btw, why not use the deployer (env::predecessor_account_id()) as an owner?

Reply:
It to be able to deploy from the factory for example. The token owner doesn't have to be the caller.